src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/Mappers/ApprenticeshipMapper.cs

```diff
@@ -59,7 +60,7 @@ public class ApprenticeshipMapper : IApprenticeshipMapper
             _academicYearValidator = academicYearValidator;
         }
 
-        public ApprenticeshipViewModel MapApprenticeship(Apprenticeship apprenticeship)
+        public ApprenticeshipViewModel MapApprenticeship(Apprenticeship apprenticeship, CommitmentView commitment)
```

Comment:
It would be good to rename this method, as it no longer only maps an apprenticeship. The equivalent method in the provider site is called MapToApprenticeshipViewModel, so perhaps we could use that name here.